exchange_server/assets/dashboards/overview.json

```diff
@@ -75,7 +86,7 @@
                 "viz": "timeseries",
                 "requests": [
                     {
-                        "q": "avg:exchange.memory.available{*}",
+                        "q": "1000000*avg:exchange.memory.available{*} by {host}",
```

Comment:
Is that because `exchange.memory.available` is in GB? If that's the case we can usually rely on metadata instead for displaying the correct unit.

Reply:
Doing so will make the unit disappear